channels/sshagent/client/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+# FreeRDP: A Remote Desktop Protocol Implementation
+# FreeRDP cmake build script
+#
+# Copyright 2012 Marc-Andre Moreau <marcandre.moreau@gmail.com>
```

Comment:
Is this correct? I assume you did write this file or am I mistaken?

Reply:
Yes: I simply copied channels/sshagent/client/CMakeLists.txt from channels/echo/client/CMakeLists.txt and replaced "echo" with "sshagent".  So I left the existing copyright line in and added my own afterwards.